packages/calcite-components/src/components/checkbox/checkbox.scss

```diff
@@ -4,16 +4,31 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-checkbox-size: Specifies the component's height and width.
+ * @prop --calcite-checkbox-box-shadow: Specifies the component's color.
+ * @prop --calcite-checkbox-box-shadow-hover: Specifies the component's color when hovered.
```

Comment:
`--calcite-checkbox-box-shadow-color-hover`

Reply:
Just sanity check here on our patterns as it has changed a few times - this is preferable to calcite-checkbox:hover { --calcite-checkbox-shadow: "whatever" },  correct?

Reply:
I renamed this one to `--calcite-checkbox-shadow-color-hover`

Reply:
> Just sanity check here on our patterns as it has changed a few times - this is preferable to calcite-checkbox:hover { --calcite-checkbox-shadow: "whatever" }, correct?

I believe so, but I'll let @alisonailea confirm. This approach is more consistent and intuitive than the previous one. It's also not mentioned as an example in the latest Component Tokens wiki.

Reply:
Agreed.